addons/controllers/calico/calicoconfig_controller.go

```diff
@@ -47,42 +48,38 @@ type CalicoConfigReconciler struct {
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
 func (r *CalicoConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Log = r.Log.WithValues("CalicoConfig", req.NamespacedName)
+	log := r.Log.WithValues("CalicoConfig", req.NamespacedName)
```

Comment:
Any reason to make this change? Cause as far as I know, having this assignment to ```r.Log``` would add those values to the logger instance of the reconciler, for better tracking the log messages.

Reply:
you are right this is unecesary change. will revert